concrete/src/Updater/Migrations/Migrations/Version20180622192332.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Concrete\Core\Updater\Migrations\Migrations;
+
+use Concrete\Core\Job\Job;
+use Concrete\Core\Job\Set;
+use Concrete\Core\Updater\Migrations\AbstractMigration;
+use Concrete\Core\Updater\Migrations\RepeatableMigrationInterface;
+use Doctrine\DBAL\Schema\Schema;
+
+
+class Version20180622192332 extends AbstractMigration implements RepeatableMigrationInterface
+{
+
+    public function upgradeDatabase()
+    {
+        $this->createSinglePage('/dashboard/system/registration/deactivation', 'User Deactivation Settings');
```

Comment:
`t('User Deactivation Settings')`?

Reply:
Nope: in the database the single page names are always stored in English: we call t() when we display them.

Reply:
You are right, and I think this is going wrong in a lot of add-ons that are submitted / approved to the marketplace :see_no_evil:

Reply:
Ok, so basically in add-ons the t-strings _should_ be used to install single pages, but in migrations not?

Reply:
>  in add-ons the t-strings *should* be used to install single pages

In add-ons, there 
If the translatable strings are present in other places (for instance, inside other t() calls in other files, or in CIF xml files), the install method of add-ons *may* use `t()` calls (since the resulting strings are the same).
If the single page names are not present anywhere else, yes, they *should* use t() calls, so that the translatable strings extractor can be aware of those strings.